volttrontesting/platform/test_certs_utils.py

```diff
@@ -0,0 +1,236 @@
+import logging
+import subprocess
+from collections import namedtuple
+from pathlib import Path
+from typing import List
+from typing import Union
+
+PathStr = Union[Path, str]
+
+__all__ = ['TLSRepository']
+
+_log = logging.getLogger(__name__)
+
+
+def execute_command(cmds, env=None, cwd=None, logger=None, err_prefix=None) -> str:
```

Comment:
just use this from the volttron directories...rather than copying it here.

this should be in one of the utils directories do a search for def execute_command and it should come up.  Note from the other repo I didn't have access to volttron's namespace.